src/crimson/osd/ops_executer.cc

```diff
@@ -693,25 +693,30 @@ OpsExecuter::execute_osd_op(OSDOp& osd_op)
       return backend.getxattr(os, osd_op);
     });
   case CEPH_OSD_OP_CREATE:
-    return do_write_op([&osd_op] (auto& backend, auto& os, auto& txn) {
+    return do_write_op([&osd_op] (auto& backend, auto& os,
+				  auto& txn, auto& osd_op_params) {
```

Comment:
`osd_op_params` is unused so maybe just `(...) txn, auto&) {`?
Alternatively we could introduce `do_write_op_with_params` (or something with a better name).

This note applies to other the `cases` as well.